src/raftstore/coprocessor/split_observer.rs

```diff
@@ -68,20 +117,33 @@ impl AdminObserver for SplitObserver {
         ctx: &mut ObserverContext,
         req: &mut AdminRequest,
     ) -> CopResult<()> {
-        if req.get_cmd_type() != AdminCmdType::Split {
-            return Ok(());
+        match req.get_cmd_type() {
+            AdminCmdType::Split => {
+                return Err(box_err!(
```

Comment:
So when upgrading exist cluster, the new version TiKV will can't propose split request?